src/test/groovy/edu/ie3/datamodel/io/source/sql/SqlWeatherSourcePsdmIT.groovy

```diff
@@ -37,10 +38,10 @@ class SqlWeatherSourcePsdmIT extends Specification implements WeatherSourceTestH
 
 	def setupSpec() {
 		// Copy sql import script into docker
-		MountableFile sqlImportFile = MountableFile.forClasspathResource("/testcontainersFiles/sql/psdm/weather.sql")
-		postgreSQLContainer.copyFileToContainer(sqlImportFile, "/home/weather.sql")
+		MountableFile sqlImportFile = getMountableFile("weather/weather_psdm.sql")
+		postgreSQLContainer.copyFileToContainer(sqlImportFile, "/home/weather_psdm.sql")
```

Comment:
Would you mind renaming the file to `weather_cosmo.sql`? We have not fully introduced it everywhere, but we agreed on orienting the naming rather at the model it stems from than the model it is represented in.